pkgs/development/ocaml-modules/gstreamer/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildDunePackage, fetchFromGitHub, dune-configurator, pkg-config, glib, gst_all_1 }:
+
+buildDunePackage rec {
+  pname = "gstreamer";
+  version = "0.3.1";
+
+  useDune2 = true;
+
+  src = fetchFromGitHub {
+    owner = "savonet";
+    repo = "ocaml-gstreamer";
+    rev = "v${version}";
+    sha256 = "0y8xi1q0ld4hrk96bn6jfh9slyjrxmnlhm662ynacp3yzalp8jji";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ dune-configurator ];
+  propagatedBuildInputs = [ glib.dev gst_all_1.gstreamer.dev gst_all_1.gst-plugins-base ];
+
+  NIX_CFLAGS_COMPILE = [
```

```suggestion
  CFLAGS_COMPILE = [
```
Please only overwrite NIX_* variables if there is no other way because this overwrites and not appends.